WordPress/Classes/Utility/Migrations/20-21/SafeReaderTopicToReaderTopic.m

```diff
@@ -8,36 +6,11 @@ @implementation SafeReaderTopicToReaderTopic
 
 - (BOOL)beginEntityMapping:(NSEntityMapping *)mapping manager:(NSMigrationManager *)manager error:(NSError **)error
```

Comment:
Whoops I hadn't realized we were using Services in migrations. That makes migrations no longer backward compatible.